src/pages/Staking/index.tsx

```diff
@@ -717,7 +738,10 @@ const Staking = (): JSX.Element => {
   const claimRewardsButtonEnabled = claimableRewardAmount?.gt(ZERO_GOVERNANCE_TOKEN_AMOUNT);
 
   const unlockFirst =
-    stakedAmount?.gt(ZERO_GOVERNANCE_TOKEN_AMOUNT) &&
+    hasStakedAmount &&
+    // eslint-disable-next-line max-len
+    // `remainingBlockNumbersToUnstake !== null` is redundant because if `hasStakedAmount` is truthy `remainingBlockNumbersToUnstake` cannot be null
+    remainingBlockNumbersToUnstake !== null &&
```

Comment:
`remainingBlockNumbersToUnstake` is null when the user has not staked. So that's considered.